examples/kogitoinfra-infinispan.yaml

```diff
@@ -0,0 +1,9 @@
+#Infinispan operator should be pre-installed in namespace
```

Comment:
If you introduce the infinispan kogitoinfra as standalone yaml file, can you do the same for kafka and keycloak ?